snuba/snuba_migrations/generic_metrics/0014_sets_mv_2.py

```diff
@@ -0,0 +1,84 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import (
+    AggregateFunction,
+    Column,
+    DateTime,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+    view_name = "generic_metric_sets_aggregation_mv"
```

Comment:
The view name should encode the version otherwise you wouldn't be creating this view now since the generated SQL has an `IF EXISTS` clause.

You should change this to something like `generic_metric_sets_aggregation_mv_v2`. That way you will have both the original materialization view and the new materialization view.

Reply:
Fixed, thank you!